front/domaintype.php

```diff
@@ -0,0 +1,36 @@
+<<?php
```

Comment:
Typo in a few files:

```suggestion
<?php
```
Also in `front/domainrelation.php` and `front/domainrecordtype.php`.